### PR TITLE
feat(profiling): add architecture column to profiles_local table

### PR DESCRIPTION
**Pull request by viglia**
This PR adds a new `architecture` column to the `profiles_local` table.

`architecture` is a string describing the architecture of the CPU that is used in the device that collected the specific profile.

### CODE REVIEW

**Review by evanh — APPROVED** [review]

File: snuba/migrations/snuba_migrations/profiles/0003_add_device_architecture.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+                table_name="profiles_local",
+                column_name="architecture",
+            )
+        ]
+
+    def forwards_dist(self) -> Sequence[operations.SqlOperation]:
+        return []
+
+    def backwards_dist(self) -> Sequence[operations.SqlOperation]:
+        return []
```
Comment (nikhars) [inline snuba/migrations/snuba_migrations/profiles/0003_add_device_architecture.py:38]: Is there a reason why the column isn't added to the distributed tables? In production, all queries go through the distributed tables. If you don't have the column defined in the distributed table you won't be able to access the column in the local tables

**Review by nikhars — CHANGES_REQUESTED** [review]

File: snuba/migrations/snuba_migrations/profiles/0003_add_device_architecture.py (hunk 1 above)
Comment (viglia) [reply snuba/migrations/snuba_migrations/profiles/0003_add_device_architecture.py:38]: @nikhars I was looking at a previous migration ([002](https://github.com/getsentry/snuba/blob/master/snuba/migrations/snuba_migrations/profiles/0002_disable_vertical_merge_algorithm.py)) as a reference.

Thanks for pointing that out.

**Review by Zylphrex — APPROVED** [review]
LGTM

**Review by nikhars — APPROVED** [review]

**Comment by viglia** [comment]
The migration has been applied successfully. I'm going to merge this.

**Pull request merged**